xradar/io/backends/nexrad_level2.py

```diff
@@ -1631,6 +1637,8 @@ def open_nexradlevel2_datatree(
     )
     ls_ds: list[xr.Dataset] = [sweep_dict[sweep] for sweep in sweep_dict.keys()]
     ls_ds.insert(0, xr.Dataset())
+    if title:
```

Comment:
This might be faster and more up to PEP8

```suggestion
    if title is not None:
```